content/en/continuous_integration/setup_pipelines/gitlab.md

```diff
@@ -10,6 +10,17 @@ further_reading:
       text: "Troubleshooting CI"
 ---
 
+## Compatibility
```

Comment:
There's a note in this page that says:

> Note: Only datadoghq.com is currently supported.

2 things:

1. Maybe it would be better to move that note to the Compatibility section?
2. Is that really a limitation? The JUnit uploader doc says everything is supported (`Possible values: datadoghq.com, datadoghq.eu or us3.datadoghq.com`)

Reply:
Because we are not deployed anywhere other than us1, we should really remove all mentions to this until we do support something else, and then only document the ones that we support